### DISCUSSION TITLE
Supabase Concurrent connections pricing

### DISCUSSION THREAD

**Discussion by christianr11** [supabase/supabase#27146] [Questions] [answered]
I checked the pricing for concurrent connection which is $10 per 1,000 users. Does this mean the total number of users who are currently online and connected to the database simultaneously?

For example: If we have 10,000 users active at the same time, does this imply that I will pay $10 per 1,000 active users? Thus, it will cost us $100 for just one day?

We are planning to transfer around 5,000,000 users to supabase. And I would like to know the monthly cost if we use supabase realtime.

**Comment by GaryAustin1** [comment, marked as answer]
The realtime bill for the month is based on the peak connections (websockets) during the month. So if your busiest day is 10,000 then your bill for the month is $100 (less included connections).
